### PR TITLE
Assets refactoring

### PR DESCRIPTION
**Pull request by JPonte**

### CODE REVIEW

File: zmessaging/src/main/scala/com/waz/db/ZMessagingDB.scala [hunk 1]
```diff
@@ -282,9 +290,63 @@ object ZMessagingDB {
...
+      import com.waz.model.AssetData.{AssetDataDao => OldAssetDao}
+      import com.waz.service.assets2.AssetStorageImpl.{AssetDao => NewAssetDao}
+
       db.execSQL(UploadAssetDao.table.createSql)
       db.execSQL(DownloadAssetDao.table.createSql)
-      db.execSQL(AssetDao.table.createSql)
+      db.execSQL(NewAssetDao.table.createSql)
+
+      db.execSQL("ALTER TABLE Messages ADD COLUMN asset_id TEXT DEFAULT null")
+      db.execSQL("UPDATE Messages SET asset_id = id WHERE asset_id = null")
```
Comment (yeryomenkom) [inline zmessaging/src/main/scala/com/waz/db/ZMessagingDB.scala]: Wrong WHERE condition. We should check message type instead.

**Comment by JPonte** [comment]
build again

**Comment by CalumMcCall** [comment]
build again

**Comment by CalumMcCall** [comment]
Closed as we have a new PR open

**Pull request closed without merging**